documentation/assemblies/cruise-control/assembly-cruise-control-concepts.adoc

```diff
@@ -3,11 +3,16 @@
 // master.adoc
 
 [id='cruise-control-concepts-{context}']
-= Cruise Control
+= Cruise Control for cluster rebalancing
 
-You can deploy {CruiseControlProject} and access a subset of its features through {ProductName} custom resources.
+You can deploy {CruiseControlProject} to your {ProductName} cluster and access its _cluster rebalancing_ feature through custom resources.
 
-Example YAML files for deploying Cruise Control and setting optimization goals are provided in `examples/cruise-control/`.
+Cruise Control is an open source system for automating Kafka operations, including monitoring cluster workload, rebalancing a cluster based on predefined constraints, and detecting and fixing anomalies, among others. 
```

Comment:
Not sure we need _among others_ if we say _including_

Reply:
Good point. Changed to "such as" and removed "among other".